ott/core/initializers.py

```diff
@@ -278,6 +285,232 @@ def init_dual_a(
     return f_u
 
 
+@jax.tree_util.register_pytree_node_class
+class FixedGeometryMetaOTInitializer(DefaultInitializer):
```

Comment:
yes, maybe a bit too long, I agree. But it's more about thinking how this will evolve, I am thinking there won't be a single `MetaOTInitializer`. Maybe `MetaInitializer`? `FixGeomMetaInitializer`?

Reply:
Hmm, we could keep `MetaOTInitializer` (or `MetaInitializer`) with a required geometry argument for now and then if it makes sense in the future we can make the geometry optional and handle the fixed vs varying geometries internally. I think it could be done in a backwards-compatible way. From the external interface here, the varying geometries would just require that the associated geometry is passed along with the probabilities, i.e. in `update` and the model, and otherwise we assume those won't take the geometry as the code is doing now. (I also don't immediately know how to nicely handle batches/vmaps over geometries)

Reply:
I think MetaOT makes sense in the context of the paper, but here in this context it's pretty clear this is OT :) So if we must shorten, probably `MetaInitializer` makes more sense